plugin/evm/vm_test.go

```diff
@@ -748,20 +739,12 @@ func TestIssueAtomicTxs(t *testing.T) {
 	logs, err := filterAPI.GetLogs(context.Background(), filters.FilterCriteria{
 		BlockHash: &blockHash,
 	})
-	if err != nil {
-		t.Fatal(err)
-	}
-	if len(logs) != 0 {
-		t.Fatalf("Expected log length to be 0, but found %d", len(logs))
-	}
-	if logs == nil {
-		t.Fatal("Expected logs to be non-nil")
-	}
+	require.NoError(err)
+	require.Empty(logs, "Expected logs to have length 0")
+	require.NotNil(logs, "Expected logs to be non-nil")
```

Comment:
I don't think we need to verify the logs

Reply:
yeah, these were copied from the existing test, removing.